pkg/client/clientset/versioned/fake/register.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var scheme = runtime.NewScheme()
 var codecs = serializer.NewCodecFactory(scheme)
-var parameterCodec = runtime.NewParameterCodec(scheme)
```

Comment:
These changes were missed as part of upgrading dependencies. Let's put them in a separate PR, or at least a separate commit in this PR. I'll put up a PR for them.

Reply:
https://github.com/openshift/hive/pull/1125

Reply:
Removed these changes, thanks